locale/foreman.pot

```diff
@@ -15392,7 +15492,7 @@ msgstr ""
 
 #:
 #: ../webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/PowerStatus/actions.js:13
-msgid "Power has been set to \"%s\" successfully"
+msgid "Power has been set to \\\"%s\\\" successfully"
```

Comment:
I wonder if this is correct.

Reply:
So it's this line:
https://github.com/theforeman/foreman/blob/38e2c55c22cec59ff1458cd4fb1e7022073a3a76/webpack/assets/javascripts/react_app/components/HostDetails/DetailsCard/PowerStatus/actions.js#L13
There are no backslashes in there, so I question where this come from.

I don't know if this really fixing an escaping issue that already existing, or if there's some duplicate escaping bug that was introduced somewhere. I'll try to dig further.

Reply:
I'm a little confused what to look into here. The original string uses single quotes and double quotes on the inside. Why are they both getting swapped to double quotes?

Reply:
https://github.com/webhippie/gettext_i18n_rails_js/commit/77743bc13b8ec79eba14165c442e1aa89ba1d04e suggests this is intentional, but it would be nice to test it out end to end.

I see there was a French translation for the old string, so you can try to update this PR with the newly translated string, wait for Packit to be built and try it out in the UI.

It's just a few strings, so I won't hold it back on this, but if we can figure this out while Pulp is getting unblocked that would be nice.

Reply:
I'm confused. Does this PR not already update the string for other languages?

Reply:
Yes and no. This PR does multiple things.

It updates the template (`.pot`). This is the file with source strings. They're never supposed to be translated, but instead are used to update the `.po` files which exist for each language. So the resulting `.po` files are a combination of the `.pot` file as well as the translations from Transifex.

The problem is that Transifex pulls its strings from git develop, so it can't know about updated strings until this is merged. Since source string changed, translators can't have translated it, so it's empty.

Perhaps at some point we can be smarter in finding these minimally changed strings, but in practice I'd expect it's rare. How do you know it's really the same?

Does that help clarify it a bit?

Reply:
ok that makes sense